arboretum/permissions/README.md

```diff
@@ -95,6 +95,78 @@ day.
   from arboretum.permissions.fetchers.github.fetch_org_collaborators import GithubOrgCollaboratorsFetcher
   ```
 
+### Organization Integrity (Repository Forks)
+
+* Class: [GithubRepoForksFetcher][gh-org-fetcher]
+* Purpose: Writes the repository forks in Github organizations to the evidence locker. This fetcher class is only meant for use with Github or Github Enterprise organizations.
+* Behavior: For each Github organization specified, an evidence file  is stored in
+the locker containing the forks for the specified repositories in the organization. The default is to
+retrieve all forks from all repositories in each specified Github organization.  TTL is set to 1
+day.
+* Configuration elements:
+  * `org.permissions.org_integrity.orgs`
+     * Required
+     * List of dictionaries each containing Github organization retrieval configuration.
+        * `url`
+           * Required
+           * String in the form of `"https://github.com/my-org"` or `"https://github.<company>.com/my-org"`.
+           * Use to define the Github organization url to use.
+        * `repos`
+           * Optional
+           * List of strings in the form of `["my-repo", "my-other-repo"]`.
+           * Defaults to all repositories in the organization.
+           * Use if looking to filter collaborator evidence to a subset of repositories in the organization otherwise do not include.
```

Comment:
By doing this you're tying the collaborators fetcher and the forks fetcher together.  If someone chooses to run both of these fetchers in their environment then they're forced to run them for the exact same set of orgs and repos.  Is that the desired behavior?